tensornetwork/network.py

```diff
@@ -91,7 +91,7 @@ def copy(self) -> Tuple["TensorNetwork", dict, dict]:
     #TODO: add support for copying CopyTensor
     node_dict = {
         node: new_net.add_node(
-            node.tensor, name=node.name, axis_names=node.axis_names)
+            self.backend.conj(node.tensor), name=node.name, axis_names=node.axis_names)
```

Comment:
Only conjugate if conj is True

Reply:
Can you help me with the Test Cases? @Thenerdstation